### ISSUE TITLE
small gotcha - if writing to temp file and doing post_ts w/ merge, new file is overwritten

### ISSUE THREAD

**Issue by jread-usgs** [USGS-R/mda.streams#115]
They use the same naming convention and this `read_ts(download_ts(var_src=ts_path$var_src, site_name=ts_path$site_name, on_local_exists="replace"))` replaces the new file with the old

**Comment by jread-usgs** [comment]
not a priority - I will work around it, but something to be aware of

**Comment by aappling-usgs** [comment]
the commit referenced above (442bc31) has what i think should be a fix. do you have a ready way to test those changes? i haven't done testing. i also made a full new PR just now, but that has tons of changes in it, so you might just want to pull in the changes in this one file.

**Comment by aappling-usgs** [comment]
should be fixed in PR #116 with these lines:
```r
pre_merge_dir <- file.path(ts_path$dir_name, "pre_merge_temp")
dir.create(pre_merge_dir, showWarnings=FALSE)
ts_old <- read_ts(download_ts(var_src=ts_path$var_src, site_name=ts_path$site_name, folder=pre_merge_dir, on_local_exists="replace"))
```
